scripts/enable.sh

```diff
@@ -77,11 +51,11 @@ else
     compose_up="false"
 fi
 
-curl -L https://github.com/docker/compose/releases/download/1.1.0-rc2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+curl -L https://github.com/docker/fig/releases/download/1.1.0-rc2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
 chmod +x /usr/local/bin/docker-compose
 
 if [ "$compose_up" != "false" ]; then
-    echo "composing:"
+    log "composing:"
     echo $compose_up | yaml_dump
     mkdir -p /home/$azureuser/compose
```

Comment:
Agree. I also asked Jeff at the time to make pulling the compose exe optional/dependent on whether there was compose content on the theory that is less like the whole script will fail or be slow due to network weather.